### PR TITLE
chore: docs updates

### PR DESCRIPTION
**Pull request by sr-remsha**

### CODE REVIEW

File: docs/user-guide.md [hunk 1]
```diff
@@ -184,6 +193,22 @@ To replay a conversation:
 
 The replayed conversation is displayed as a new conversation with the `[Replay]` tag in the list of your conversations. 
 
+### Parameterized Replay
+
+If you want to make your own chats or stories with custom variables and share them with your team, you can use a feature called Parameterized Replay. Use it to set up a chat or a story that others can follow, but provide their own details or answers to create a personalized experience. This makes chat a much more useful tool for sharing information and collaboration.
+
+How it works: 
+
+Let's create a simple Travel Guide conversation, which will take a user though several stepts to clarify information and provide the answer.
```
Comment (VolhaBazhkova) [inline docs/user-guide.md:202]: misprint in 'steps'

File: docs/user-guide.md [hunk 2]
```diff
@@ -184,6 +193,22 @@ To replay a conversation:
...
+
+1. Start by [creating a special prompt](#create-1) with [variables](#variables). Use templates (this `{{VariableName|DefaultValue}}` or this `{{VariableName}}`) to add variables into your prompt with or without default values. For example: `I'd like to travel to {{country|Japan}}. Could you please suggest {{num-attractions|10}} of the best attractions? I will be there for {{num-days}}. Thank you.`
+2. To use it, type `/` in the conversation text input area and select your prompt. You will be prompted to provide your inputs for variables in a pop-up window. Enter your inputs and click `Submit`. The prompt is now populated with your inputs in the conversation text box. Click `Play` to start.
```
Comment (VolhaBazhkova) [inline docs/user-guide.md:205]: Do you mean 'Send a message to a chat?' 'Start replay' button exists in Replay mode only.

File: docs/user-guide.md [hunk 3]
```diff
@@ -157,14 +158,22 @@ To import JSON or a zip file with conversations, click the **Import conversation
...
-To delete a single conversation, on the left panel, in the conversation contextual menu, select **Delete** and confirm your action.
+* To delete a single conversation, on the left panel, in the conversation contextual menu, select **Delete** and confirm your action.
+* To delete all conversations, at the bottom of the left panel, click the **Delete all conversations** icon.
 
 ![](./img/Delete_conv.png)
 
-To delete all conversations, at the bottom of the left panel, click the **Delete conversations** icon.
- 
+
+* To delete selected conversations, you can use either **Select all** button and choose the conversations you want to remove
```
Comment (VolhaBazhkova) [inline docs/user-guide.md:169]: Agreed via chat in Teams to rephrase a little bit

**Review by VolhaBazhkova — CHANGES_REQUESTED** [review]
several updates are needed

File: docs/user-guide.md (hunk 1 above)
Comment (sr-remsha) [reply docs/user-guide.md]: fixed

File: docs/user-guide.md (hunk 2 above)
Comment (sr-remsha) [reply docs/user-guide.md]: fixed

File: docs/user-guide.md (hunk 3 above)
Comment (sr-remsha) [reply docs/user-guide.md]: the section war rephrased

**Review by VolhaBazhkova — APPROVED** [review]

**Pull request merged**